### ISSUE TITLE
R: How to obtain the probabilites using objective = multi:softmax

### ISSUE THREAD

**Issue by jakob-r** [dmlc/xgboost#101]
Using this little **R**-example I would like to obtain the probabilities for each class and not only the label.

``` r
library(xgboost)
d = matrix(runif(300), nrow = 100)
y = sample(0:2, size = 100, replace = T)
d = xgb.DMatrix(data = d, label = y)
m = xgb.train(data = d, nrounds = 1, objective = "multi:softmax", num_class = 3)
p = predict(m, newdata = d)
```
